samples/snippets/src/main/java/aiplatform/BatchReadFeatureValuesSample.java

```diff
@@ -0,0 +1,112 @@
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public class BatchReadFeatureValuesSample {
+
+  public static void main(String[] args)
+      throws IOException, InterruptedException, ExecutionException, TimeoutException {
+    // TODO(developer): Replace these variables before running the sample.
+    String project = "YOUR_PROJECT_ID";
+    String featurestoreId = "YOUR_FEATURESTORE_ID";
+    String entityTypeId = "YOUR_ENTITY_TYPE_ID";
+    String inputCsvFile = "YOU_INPUT_CSV_FILE";
+    String destinationTableUri = "YOUR_DESTINATION_TABLE_URI";
+    String location = "us-central1";
+    String endpoint = "us-central1-aiplatform.googleapis.com:443";
+    int timeout = 300;
+    batchReadFeatureValuesSample(project, featurestoreId, entityTypeId, inputCsvFile,
+        destinationTableUri, location, endpoint, timeout);
+  }
+
+  static void batchReadFeatureValuesSample(String project, String featurestoreId,
+      String entityTypeId, String inputCsvFile, String destinationTableUri, String location,
+      String endpoint, int timeout)
+      throws IOException, InterruptedException, ExecutionException, TimeoutException {
+    FeaturestoreServiceSettings featurestoreServiceSettings =
+        FeaturestoreServiceSettings.newBuilder().setEndpoint(endpoint).build();
+
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (FeaturestoreServiceClient featurestoreServiceClient =
+        FeaturestoreServiceClient.create(featurestoreServiceSettings)) {
+
+      List<EntityTypeSpec> entityTypeSpecs = new ArrayList<>();
+
+      List<String> ids = new ArrayList<>();
+      ids.add("*");
+      FeatureSelector featureSelector = FeatureSelector.newBuilder()
+          .setIdMatcher(IdMatcher.newBuilder().addAllIds(ids).build()).build();
+      EntityTypeSpec entityTypeSpec = EntityTypeSpec.newBuilder().setEntityTypeId(entityTypeId)
+          .setFeatureSelector(featureSelector).build();
+      entityTypeSpecs.add(entityTypeSpec);
+
+      BigQueryDestination bigQueryDestination =
+          BigQueryDestination.newBuilder().setOutputUri(destinationTableUri).build();
+      GcsSource gcsSource = GcsSource.newBuilder().addUris(inputCsvFile).build();
+      BatchReadFeatureValuesRequest batchReadFeatureValuesRequest =
+          BatchReadFeatureValuesRequest.newBuilder()
+              .setFeaturestore(FeaturestoreName.of(project, location, featurestoreId).toString())
+              .setCsvReadInstances(CsvSource.newBuilder().setGcsSource(gcsSource))
+              .setDestination(
+                  FeatureValueDestination.newBuilder().setBigqueryDestination(bigQueryDestination))
+              // .addAllPassThroughFields(passThroughFieldsList)
```

Comment:
should we include this as well, if possible ?

Reply:
removing that as there are not passThroughFields for the movie prediction example.